src/terraform/providers/terraform-provider-avere/resource_vfxt.go

```diff
@@ -1143,16 +1150,24 @@ func createCoreFilers(coreFilersToAdd map[string]*CoreFiler, averevfxt *AvereVfx
 }
 
 func startFixedQuotaPercent(corefilers []*CoreFiler, averevfxt *AvereVfxt) error {
-	if isRequired := isFixedQuotaRequired(corefilers); isRequired {
+	if smallestQuota, isRequired := isFixedQuotaRequired(corefilers); isRequired {
 		if err := averevfxt.CreateCustomSetting(QuotaCacheMoveMax); err != nil {
 			return fmt.Errorf("ERROR: failed to apply custom setting '%s': %s", QuotaCacheMoveMax, err)
 		}
+
+		if err := averevfxt.CreateCustomSetting(fmt.Sprintf(QuotaDivisorFloor, smallestQuota)); err != nil {
+			return fmt.Errorf("ERROR: failed to apply custom setting '%s': %s", QuotaCacheMoveMax, err)
+		}
+
+		if err := averevfxt.CreateCustomSetting(QuotaMaxMultiplierForInvalidatedMassQuota); err != nil {
+			return fmt.Errorf("ERROR: failed to apply custom setting '%s': %s", QuotaCacheMoveMax, err)
```

Comment:
Error should print QuotaMaxMultiplierForInvalidatedMassQuota

Reply:
fixed